external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaSinkSuite.scala

```diff
@@ -74,21 +78,28 @@ abstract class KafkaSinkSuiteBase extends QueryTest with SharedSparkSession with
   }
 }
 
-class KafkaSinkStreamingSuite extends KafkaSinkSuiteBase with StreamTest {
+abstract class KafkaSinkStreamingSuiteBase extends KafkaSinkSuiteBase {
   import testImplicits._
 
-  override val streamingTimeout = 30.seconds
+  protected val streamingTimeout = 30.seconds
+
+  // TODO: this is set to "Int" since ContinuousMemoryStream cannot deal with String
```

Comment:
I might file another issue soon for fixing ContinuousMemoryStream to support String. There seems to some hurdles to fix that, so I just defer to the another issue. Please let me know if we want to fix it altogether.

Reply:
SPARK-29642 (#26300)

When #26300 is merged, I'll apply TODO describing here and remove TODO.

Reply:
SPARK-29642 is resolved - I'm making the change.